plugins/inputs/modbus/modbus.go

```diff
@@ -441,6 +497,11 @@ func (m *Modbus) collectFields(acc telegraf.Accumulator, timestamp time.Time, ta
 	}
 }
 
+// Implement the logger interface of the modbus client
+func (m *Modbus) Printf(format string, v ...interface{}) {
+	m.Log.Debugf(format, v...)
```

Comment:
These won't show up unless the user also has the --debug command line option.

Reply:
Right, therefore the text for the option enabling this output states

>  ## Trace the connection to the modbus device as debug messages

suggesting that you need both, `--debug` and the `debug_connection` option enabled.

Do you have a suggestion on how to improve either the text or the handling? As discussed elsewhere, we might also implement a telegraf `trace` level, but IIRC the decision was to stick with debug and individual options...

Reply:
Let's add something to the readme to remind users that when they enable debug_connnection, they need to enable debug. Maybe a link to https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#agent. That's good enough for me